generators/base-application/priorities.cjs

```diff
@@ -0,0 +1,138 @@
+const PREPARING_EACH_ENTITY_QUEUE = `${QUEUE_PREFIX}${PREPARING_EACH_ENTITY}`;
+
+const PREPARING_EACH_ENTITY_FIELD = 'preparingEachEntityField';
+const PREPARING_EACH_ENTITY_FIELD_QUEUE = `${QUEUE_PREFIX}${PREPARING_EACH_ENTITY_FIELD}`;
+
+const PREPARING_EACH_ENTITY_RELATIONSHIP = 'preparingEachEntityRelationship';
+const PREPARING_EACH_ENTITY_RELATIONSHIP_QUEUE = `${QUEUE_PREFIX}${PREPARING_EACH_ENTITY_RELATIONSHIP}`;
+
+const POST_PREPARING_EACH_ENTITY = 'postPreparingEachEntity';
+const POST_PREPARING_EACH_ENTITY_QUEUE = `${QUEUE_PREFIX}${POST_PREPARING_EACH_ENTITY}`;
+
+const WRITING_ENTITIES = 'writingEntities';
+const WRITING_ENTITIES_QUEUE = `${QUEUE_PREFIX}${WRITING_ENTITIES}`;
+
+const POST_WRITING_ENTITIES = 'postWritingEntities';
+const POST_WRITING_ENTITIES_QUEUE = `${QUEUE_PREFIX}${POST_WRITING_ENTITIES}`;
+
+const CUSTOM_PRIORITIES = [
+  {
+    priorityName: CONFIGURING_EACH_ENTITY,
+    queueName: CONFIGURING_EACH_ENTITY_QUEUE,
+    before: LOADING_ENTITIES,
+    skip: true,
+  },
+  {
+    priorityName: LOADING_ENTITIES,
+    queueName: LOADING_ENTITIES_QUEUE,
+    before: PREPARING_EACH_ENTITY,
+    skip: true,
+  },
+  {
+    priorityName: PREPARING_EACH_ENTITY,
+    queueName: PREPARING_EACH_ENTITY_QUEUE,
+    before: PREPARING_EACH_ENTITY_FIELD,
+    skip: true,
+  },
+  {
+    priorityName: PREPARING_EACH_ENTITY_FIELD,
+    queueName: PREPARING_EACH_ENTITY_FIELD_QUEUE,
+    before: PREPARING_EACH_ENTITY_RELATIONSHIP,
+    skip: true,
+  },
+  {
+    priorityName: PREPARING_EACH_ENTITY_RELATIONSHIP,
+    queueName: PREPARING_EACH_ENTITY_RELATIONSHIP_QUEUE,
+    before: POST_PREPARING_EACH_ENTITY,
+    skip: true,
+  },
+  {
+    priorityName: POST_PREPARING_EACH_ENTITY,
+    queueName: POST_PREPARING_EACH_ENTITY_QUEUE,
+    before: DEFAULT,
+    skip: true,
+  },
+  {
+    priorityName: WRITING_ENTITIES,
+    queueName: WRITING_ENTITIES_QUEUE,
+    before: TRANSFORM,
+    skip: true,
+  },
+  {
+    priorityName: POST_WRITING_ENTITIES,
+    queueName: POST_WRITING_ENTITIES_QUEUE,
+    before: PRE_CONFLICTS,
+    skip: true,
+  },
+].reverse();
+
+const ENTITY_QUEUES = {
+  CONFIGURING_EACH_ENTITY_QUEUE,
+  LOADING_ENTITIES_QUEUE,
+  PREPARING_EACH_ENTITY_QUEUE,
+  PREPARING_EACH_ENTITY_FIELD_QUEUE,
+  PREPARING_EACH_ENTITY_RELATIONSHIP_QUEUE,
+  POST_PREPARING_EACH_ENTITY_QUEUE,
+  WRITING_ENTITIES_QUEUE,
+  POST_WRITING_ENTITIES_QUEUE,
+};
+
+const ENTITY_PRIORITY_NAMES = {
+  CONFIGURING_EACH_ENTITY,
+  LOADING_ENTITIES,
+  PREPARING_EACH_ENTITY,
+  PREPARING_EACH_ENTITY_FIELD,
+  PREPARING_EACH_ENTITY_RELATIONSHIP,
+  POST_PREPARING_EACH_ENTITY,
+  WRITING_ENTITIES,
+  POST_WRITING_ENTITIES,
+};
+
+const PRIORITY_NAMES_LIST = [
+  PRIORITY_NAMES.INITIALIZING,
+  PRIORITY_NAMES.PROMPTING,
+  PRIORITY_NAMES.CONFIGURING,
+  PRIORITY_NAMES.COMPOSING,
+  PRIORITY_NAMES.LOADING,
+  PRIORITY_NAMES.PREPARING,
+  CONFIGURING_EACH_ENTITY,
+  LOADING_ENTITIES,
+  PREPARING_EACH_ENTITY,
```

Comment:
The EACH term is a bit confusing, it could mean that it will modify EACH Entities ^^.

Removing the EACH looks to be more meaningful (meaning that we are focusing on one specific entity).
Am I wrong with the meaning? If it really means EACH, I would go for PREPARING_ENTITIES.

Reply:
I think we can rename, but let's do it in another PR.